index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+
+    <link rel="stylesheet" href="css/normalize.css">
+    <link rel="stylesheet" href="css/style.css">
+    <link href='https://fonts.googleapis.com/css?family=Courgette|Special+Elite|Arvo' rel='stylesheet' type='text/css'>
+
+    <title>Where in the World Quiz</title>
+</head>
+
+<body>
+
+    <header>
+        <h1>Where in the World are You?</h1>
+    </header>
+
+    <section class="introduction">
+        <h2 class="top-rule">Can you figure out where in the world each photo was taken?</h2>
+        <h2>Study the photos and look for hints!</h2>
+        <button id="start" class="startQuiz">Start Jetsetting!</button>
+    </section>
+
+    <section class="questions">
+        <div id="question-wrapper">
+            <h2 id="addQuestionText"></h2>
+        </div>
+        <div id="image"><img src="images/Montmartre.jpg" />
+        </div>
+        <div class="choices-list">
+            <ul id="userChoices" class="choices">
+
+
+            </ul>
+        </div>
+        <div class="button-placer">
+            <button id="submit" class="guess">Guess</button>
+        </div>
+
+        <h3 id="question-counter" class="question-tracker" id="count"></h3>
```

Comment:
very likely it would be the second because the first one would be set, then the second one would overwrite it. It's possible that _some_ browsers might recognize both, but likely not all